### PR TITLE
Implemented `DJANGO_EASY_AUDIT_PROPAGATE_EXCEPTIONS` setting

### PR DESCRIPTION
**Pull request by dferens**
Hi @jheld ,

This is a PR for https://github.com/soynatan/django-easy-audit/issues/247 , let me know if there's something I need to change.

### CODE REVIEW

**Review by mschoettle — CHANGES_REQUESTED** [review]
I am not a maintainer so am providing community feedback. Added a couple of comments.

I like this extra setting. Every once in a while I see some exceptions in pytest output that are suppressed.

File: easyaudit/signals/auth_signals.py [hunk 1]
```diff
@@ -19,8 +20,9 @@ def user_logged_in(sender, request, user, **kwargs):
                 'user_id': getattr(user, 'id', None),
                 'remote_ip': request.META[REMOTE_ADDR_HEADER]
             })
-    except:
-        pass
+    except Exception:
+        if settings.DEBUG and DEBUG_SIGNALS:
```
[the remark this answers is not in the document]
Comment (jheld) [reply easyaudit/signals/auth_signals.py:24]: I'm open to encapsulating the logic more explicitly like that.

[the remark this answers is not in the document]
Comment (dferens) [reply easyaudit/signals/auth_signals.py]: done in 6c6b16ffb890b9ff21226caa7a4a7214a8f42ac5

File: README.md [hunk 2]
```diff
@@ -99,7 +99,11 @@ Below are some of the settings you may want to use. These should be defined in y
   this is necessary in order to keep database atomicity concerns in check during signal handlers.
 
   To clarify, this is only _truly_ necessary for the model signals.
-  
+
+* `DJANGO_EASY_AUDIT_DEBUG_SIGNALS`
+
+  Default is `False`. When set to `True` and Django's `DEBUG` is enabled, easyaudit will propagate exceptions occurred in own signal handlers. This is useful, unless you are using `DEBUG` in production.
```
[the remark this answers is not in the document]
Comment (dferens) [reply README.md:105]: @jheld need your input on this

[the remark this answers is not in the document]
Comment (jheld) [reply README.md:105]: Is the argument for having 2 variables (at least in the short term)? If so, I like that. We can separately issue a pending deprecation warning for the older style settings variable at any point in time (if we find that the original setting has a poor name and the behavior itself can be more configurable in nature).

**Comment by dferens** [comment]
Hi @jheld , I reworked PR for a single setting and added test.

**Review by mschoettle — COMMENTED** [review]
Looks great, thanks!

There are a few lines (`raises e`) where no `should_propagate_exceptions()` is called first. Are those checks missing there?

**Comment by dferens** [comment]
> There are a few lines (`raises e`) where no `should_propagate_exceptions()` is called first. Are those checks missing there?

I only modified top-level try/catch blocks, not sure if we need to change inner ones.

**Comment by dferens** [comment]
Hi, how can we proceed with this?

**Review by jheld — APPROVED** [review]

**Pull request merged**